src/lib/eina/eina_inline_lock.x

```diff
@@ -0,0 +1,190 @@
+/* EINA - EFL data type library
+ * Copyright (C) 2020 Lucas Cavalcante de Sousa
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library;
+ * if not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef EINA_INLINE_LOCK_X_
+#define EINA_INLINE_LOCK_X_
+
+#ifdef _WIN32
+# ifndef WIN32_LEAN_AND_MEAN
+#  define WIN32_LEAN_AND_MEAN
+# endif
+#endif
+
+#include <errno.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include <sys/time.h>
+#include <sys/types.h>
+
+#ifdef EINA_HAVE_DEBUG_THREADS
+#include <assert.h>
```

Comment:
i'm not familiar with efl code convention, but should be a space between # and include?

Reply:
I don't have any idea, I didn't find anything about defines at EFL code conventions.